### DISCUSSION TITLE
Un tutorial muy claro y divertido

### DISCUSSION THREAD

**Discussion by NeisserMS** [skills/.github#238] [General]
Estoy feliz de seguir aprendiendo y  más cuando es fácil entender las instrucciones.

**Comment by Sithninja68** [comment]
gracias estoy totalmente de acuerdo. Sin embargo, tengo problemas para
iniciar sesión.



On Fri, May 26, 2023 at 1:05 AM Neisser Moreno ***@***.***>
wrote:

> Estoy feliz de seguir aprendiendo y más cuando es fácil entender las
> instrucciones.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/skills/discussions/238>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZ6SLWARL2RPEDVR6N5NYMLXIA23LANCNFSM6AAAAAAYPXEPWQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
Sent from Gmail Mobile
